charts/azuredisk-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.1.0-alpha"
+description: Azure disk Container Storage Interface (CSI) Storage Plugin
+name: azuredisk-csi-driver
+version: 0.1.0-alpha
```

Comment:
this is chart version, use `1.0.0`?

Reply:
I don't think `1.0.0` is ok. This does not comply with the semver specification. [Please see](https://semver.org/#how-do-i-know-when-to-release-100). `0.1.0` or `0.2.0` might be better.

Reply:
ok, then use `0.1.0`